reco_utils/dataset/criteo_dac.py

```diff
@@ -20,9 +14,18 @@
 except ImportError:
     pass  # so the environment without spark doesn't break
 
+from reco_utils.dataset.url_utils import maybe_download, remove_filepath
+from reco_utils.common.notebook_utils import is_databricks
+
+
+CRITEO_URL_FULL = "https://s3-eu-west-1.amazonaws.com/kaggle-display-advertising-challenge-dataset/dac.tar.gz"
+CRITEO_URL_SAMPLE = "http://labs.criteo.com/wp-content/uploads/2015/04/dac_sample.tar.gz"
+
+
 def load_spark_df(
     spark,
-    type='train',
+    size="sample",
+    type="train",
```

Comment:
hey @jreynolds01 a question here, do we really need to give the option for getting the test set if it doesn't have labels? If we can remove this, the code will be simpler

Reply:
I think this could be a good discussion point... currently to use the identical train and test split to other notebooks (which is needed for benchmarking/comparison), we use the same seed for splitter. If we support that from dataloader level, users don't need to worry about checking seed from the other notebooks.

Some datasets may have pre-split train and test sets in their files, but not sure if we want to give that back though.

Reply:
the issue with this test set is that we don't have the labels (correct me Jeremy if I'm wrong), so not sure in which situation we will use this in our repo

Reply:
ok, I'll assume for now that we are not using the test part for simplicity. In the future, we can change this if needed